### PR TITLE
Feature/use rxswift in example

### PR DESCRIPTION
**Pull request by raphrel**

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -82,133 +99,188 @@ final class ViewController: UIViewController {
...
+            .drive(timingLabel.rx.text)
+            .disposed(by: disposeBag)
+        
+        // Seek
+        isSliderSeeking
+            .distinctUntilChanged()
+            .skip(1) // skip init value
+            .filter { !$0 }
+            .withLatestFrom(itemDurationSubject.asObservable())
+            .filter { $0 != nil }.map { $0! }
```
Comment (JiceDessaint) [inline Example/ViewController.swift]: missing CRLF

File: Example/ViewController.swift [hunk 2]
```diff
@@ -82,133 +99,188 @@ final class ViewController: UIViewController {
...
+        
+        // Display itemUrl
+        player.rx.currentItemUrl
+            .observeOn(concurrentBackgroundScheduler)
+            .map { $0?.absoluteString }
+            .asDriver(onErrorJustReturn: nil)
+            .drive(currentItemLabel.rx.text)
+            .disposed(by: disposeBag)
+        
+        // Display dbugMessage
```
Comment (JiceDessaint) [inline Example/ViewController.swift]: // Display debugMessage

File: Example/ViewController.swift (hunk 1 above)
Comment (raphrel) [reply Example/ViewController.swift]: I dit it intentionally 
.filter { $0 != nil }.map { $0! } = unwrap

more readable for me, but maybe not

File: Example/ViewController.swift (hunk 2 above)
Comment (raphrel) [reply Example/ViewController.swift]: sure !

File: Example/ViewController.swift (hunk 1 above)
Comment (JiceDessaint) [reply Example/ViewController.swift]: Hum I see!
you can also do:
```swift 
            .flatMap { Observable.from(optional: $0) }
```
your choice. I'm ok with each

Comment (raphrel) [reply Example/ViewController.swift]: didn't know this way, thanks.

**Pull request merged**